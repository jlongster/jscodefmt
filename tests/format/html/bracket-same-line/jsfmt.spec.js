run_spec(__dirname, ["html"], { bracketSameLine: true });
