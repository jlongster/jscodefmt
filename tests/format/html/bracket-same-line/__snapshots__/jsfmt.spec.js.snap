// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`html.html - {"bracketSameLine":true} format 1`] = `
====================================options=====================================
bracketSameLine: true
parsers: ["html"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<div long_long_attribute="long_long_long_long_long_long_long_long_long_long_long_value">
text
</div>
<div long_long_attribute="long_long_long_long_long_long_long_long_long_long_long_value"></div>
<span long_long_attribute="long_long_long_long_long_long_long_long_long_long_long_value">
text
</span>
<span long_long_attribute="long_long_long_long_long_long_long_long_long_long_long_value"></span>
<img long_long_attribute="long_long_long_long_long_long_long_long_long_long_long_value"/>
<div class="a">
text
</div>
<span  class="a">
text
</span>
<img class="a"/>

=====================================output=====================================
<div
  long_long_attribute="long_long_long_long_long_long_long_long_long_long_long_value">
  text
</div>
<div
  long_long_attribute="long_long_long_long_long_long_long_long_long_long_long_value"></div>
<span
  long_long_attribute="long_long_long_long_long_long_long_long_long_long_long_value">
  text
</span>
<span
  long_long_attribute="long_long_long_long_long_long_long_long_long_long_long_value"></span>
<img
  long_long_attribute="long_long_long_long_long_long_long_long_long_long_long_value" />
<div class="a">text</div>
<span class="a"> text </span>
<img class="a" />

================================================================================
`;
